lib/chainweb/pact/send_request_body.ex

```diff
@@ -3,11 +3,11 @@ defmodule Kadena.Chainweb.Pact.SendRequestBody do
   `SendRequestBody` struct definition.
   """
 
-  alias Kadena.Types.{Command, CommandsList, PactTransactionHash, SignaturesList}
+  alias Kadena.Types.{Command, PactTransactionHash, SignaturesList}
 
   @behaviour Kadena.Chainweb.Pact.Type
 
-  @type cmds :: CommandsList.t()
+  @type cmds :: list(command())
```

Comment:
@EdwinGuayacan at line 10, `command()` does not exists. Place the type command before!

Reply:
Or write directly `list(Command.t())`!

Reply:
the `command()` type exists and is used in the `@spec` of the `extract_cmds_info` function, I'm going to move it up.

Reply:
Great